test/acceptance/features/unbindAppToService.feature

```diff
@@ -247,3 +247,129 @@ Feature: Unbind an application from a service
         * File "/bindings/spec-remove-bindings-as-files-app-sb/port" is unavailable in application pod
         * File "/bindings/spec-remove-bindings-as-files-app-sb/type" is unavailable in application pod
         * Service Binding secret is not present
+
+    Scenario: Bind without SERVICE_BINDING_ROOT set, unbind and re-bind
+        Given Generic test application is running
+        * The env var "SERVICE_BINDING_ROOT" is not available to the application
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: $scenario_id
+                annotations:
+                    "service.binding/host": "path={.spec.host}"
+            spec:
+                host: example.common
+            """
+        * Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: $scenario_id
+                application:
+                    name: $scenario_id
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        * Service Binding is ready
+        * The application env var "SERVICE_BINDING_ROOT" has value "/bindings"
+        * Service binding "$scenario_id" is deleted
+        * The application got redeployed 2 times so far
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: $scenario_id
+                application:
+                    name: $scenario_id
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then Service Binding is ready
+        * The env var "host" is not available to the application
+        * The application env var "SERVICE_BINDING_ROOT" has value "/bindings"
+        * Content of file "/bindings/$scenario_id/host" in application pod is
+            """
+            example.common
+            """
+        * The application "$scenario_id" container has only one env "SERVICE_BINDING_ROOT" set
+
+    Scenario: Bind with SERVICE_BIDNING_ROOT already set, unbind and re-bind
```

Comment:
same question as above?

Reply:
After deeper consideration, this is redundant test, will remove it, thx